### ISSUE TITLE
Language argument in byHashtag() [feature_request]

### ISSUE THREAD

**Issue by xvariable** [davidteather/TikTok-Api#87]
Dear David, 

Thank you for implementing byHashtag() and improving it to get more results. This is really helpful! Would it perhaps be possible to add a language argument to the method?

Further, would like to know how this API works. Is there any chance of getting information on all videos posted with a particular hashtag? I´m not interested in the videos, but in the comments. Is there any possibility to get them?

Best,
Julia

**Comment by davidteather** [comment]
I was just checking to see if the language argument is working, because it should. Right now I can't seem to get that working for some reason. I'll look into it over the next few days.

According to another API owner, there isn't currently any support for comments on a tiktok video. If you're able to locate a page on the desktop tiktok I can probably figure out a soltuion.

**Comment by davidteather** [comment]
I don't think this is possible. I ended up looking into it further and when you go to a url like [this](https://www.tiktok.com/trending?lang=de) (in incognito to avoid cookies) one it returns the same Tiktoks no matter what lang you pass in as a parameter.

There's a region parameter I was attempting to use, but that hasn't given me any successful results yet. Current changes are on the nightly branch.

**Comment by madnai** [comment]
I 've notice the same issue. I think the list of tiktok is based on IP address or to be more specific (on ip location). When I use proxy from USA it returns me usa tiktokers, when I use proxy from France it return france tiktokers. 
Maybe the way to make it multilanguage is to add proxy functionality to the package.

**Comment by davidteather** [comment]
I’ve come back for an update. I’ve added the language parameter option onto all the requests, but that doesn’t seem to do much. It’s there if anyone really needs it for some reason. 

The next release (hopefully in the next few days) will have proxy support.
